cms/static/js/views/settings/main.js

```diff
@@ -164,7 +209,26 @@ var DetailsView = ValidatingView.extend({
         $(e.currentTarget).attr('title', currentTimeText);
     },
     updateModel: function(event) {
+        var value;
+        var index = event.currentTarget.getAttribute('data-index');
         switch (event.currentTarget.id) {
+        case 'course-learning-info-' + index:
```

Comment:
But why is this necessary? The views themselves are updating the models when editing occurs. Don't you end up with double updates?